docs/diagrams/GoToUiSequenceDiagram.puml

```diff
@@ -0,0 +1,83 @@
+@startuml
+!include style.puml
+
+box UI UI_COLOR_T1
+participant ":TravelPlanPanel" as TravelPlanPanel UI_COLOR_T3
+participant ":MainWindow" as MainWindow UI_COLOR_T3
+end box
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":WanderLustParser" as WanderLustParser LOGIC_COLOR
+participant ":WanderLustGoToCommandParser" as WanderLustGoToCommandParser LOGIC_COLOR
+participant ":GoToCommand" as GoToCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> MainWindow : executeCommand("goto -travelplan 2")
+activate MainWindow
+MainWindow -> LogicManager : execute("goto -travelplan 2")
+activate LogicManager
+
+LogicManager -> WanderLustParser :parseCommand("goto -travelplan 2")
+activate WanderLustParser
+
+create WanderLustGoToCommandParser
+WanderLustParser -> WanderLustGoToCommandParser
+activate WanderLustGoToCommandParser
+
+WanderLustGoToCommandParser --> WanderLustParser
+deactivate WanderLustGoToCommandParser
+
+WanderLustParser -> WanderLustGoToCommandParser :parse(" -travelplan 2")
```

Comment:
There is a space in front of -tra... ? Cos I tot just "-travelplan 2"

Reply:
no theres a space..  thats why we use keyword[1] instead or [0] to access "-travelplan'"

Reply:
oo okok cos i think i didnt put space in front of mine

Reply:
Actually, I might try to change it so that there isn't so much magic number here and there, using the ArgumentTokenizer, cos you guys didn't utilise it i think?

Reply:
can use constants to name numbers?